flink-tests/src/test/java/org/apache/flink/test/streaming/runtime/BroadcastStateITCase.java

```diff
@@ -169,7 +170,10 @@ public void open(Configuration parameters) throws Exception {
 
 		@Override
 		public void processElement(Long value, KeyedReadOnlyContext ctx, Collector<String> out) throws Exception {
-			ctx.timerService().registerEventTimeTimer(timerTimestamp);
+			long currentTime = al.getAndIncrement();
+			ctx.timerService().registerEventTimeTimer(currentTime);
+			expectedTimeAndKey.put(currentTime, value);
+			expectedKeyAndTime.put(value, currentTime);
```

Comment:
What's the value of this `expectedKeyAndTime` mapping? because incrementing of the `al` counter, keys in the first map (`expectedTimeAndKey`) will always be unique and I think we can safely remove this second map.